### PR TITLE
Milestone 6: Add linters

### PR DESCRIPTION
**Pull request by odhiambo-ed**
### React capstone project - Metrics webapp

In this milestone, I did the following:

  - [ ] Added a Home page.  
  - [ ] Added a Details page. 
  - [ ]  Created unit tests for pure functions in your Redux code.  
  - [ ] Displayed data from API.  Added Style.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,73 +1,87 @@
...
+- LinkedIn: [Edward Odhiambo](https://www.linkedin.com/in/edward-odhiambo-6a462a21b/)
 
-## Acknowledgments
+## Acknowledgements
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+* [ReactJS Organization](https://reactjs.org/) for the amazing [Documentation](https://reactjs.org/docs/getting-started.html) on React.
+* [Redux Organization](https://redux.js.org/) for the amazing [Redux Essentials Tutorial](https://redux.js.org/tutorials/essentials/part-1-overview-concepts) on React.
```
Comment (bimbolabuari) [inline README.md]: Kindly give appropriate credit to Nelson Sakwa the author of the original design as required.

**Review by bimbolabuari — CHANGES_REQUESTED** [review]
Hi @white3d, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Deploys the application using Netlify
- The project contains a `Home` page and a `Detail` page.
- The project is a single page application (SPA) using `react-router-dom` to serve each page without reloading.

## Required Changes ♻️
- [ ] Kindly record a video presenting the features of the project you built. It is a video with a maximum length of 5 minutes
- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1,73 +1,87 @@
...
 
-## 📝 License
+## Show your support
```
Comment (Mihndim2020) [inline README.md:88]: - [ ] As requested by the previous code reviewer, which is a [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md) for this project, kindly add "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
## Hi @white3d, 

Great work on making the changes requested by a previous reviewer 👏 👏 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

stay-strong-youre-almost-there


 # Status: Requires Changes ♻️
_Check the comments below and under the review._

Please do consider, as requested by the previews reviewer:
- [ ] To record a video presenting the features of the project, the video should;
   - should maximum length of the video is 5 minutes,
   - should contain a clear and objective description of the project in professional English. The topic and purpose of the project are made clear,
   - should contain a demo of the project, highlighting all the project features,
   - should contain a highlight about a piece of the code that you find interesting or is proud of making,
- [ ] Add a link to the video either in your `Readme file` of `PR summary`.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -79,6 +80,7 @@ Feel free to check the [issues page](../../issues)
 
 ## Acknowledgements
 
+* Original design idea by [Nelson Sakwa on Behance](https://www.behance.net/sakwadesignstudio)
```
Comment (topeogunleye) [inline README.md:83]: Appropriate credit to Nelson Sakwa in your Acknowledgments section, as required by the Creative Commons License :heavy_check_mark:

Comment (topeogunleye) [inline README.md:83]: Appropriate credit to Nelson Sakwa in your Acknowledgments section, as required by the Creative Commons License :heavy_check_mark:

File: README.md [hunk 4]
```diff
@@ -19,6 +19,7 @@ This is a React app that fetch metrics from Narrativa API. It is about building
 
 
 ## [Live Demo - Netlify](https://aesthetic-cupcake-19dcf4.netlify.app)
+## [Live Demo Video - Loom](https://www.loom.com/share/ff5b88499f064236867a193d28c8f8e2)
```
Comment (topeogunleye) [inline README.md:22]: Great Job with adding your loom video describing your project :heavy_check_mark:

**Review by topeogunleye — APPROVED** [review]
### **STATUS: APPROVED  ✔️** 
------
Hi @white3d,  

Your project is complete! Great job for nailing it right on the first attempt 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟
congratulations

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Followed Gitflow :heavy_check_mark:
- PR has a good title and description :heavy_check_mark:

> ## [Optional] suggestions:
> * Nothing to mention.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**